app/src/main/java/org/oppia/app/help/HelpViewModel.kt

```diff
@@ -1,9 +1,29 @@
 package org.oppia.app.help
 
-import androidx.lifecycle.ViewModel
-import org.oppia.app.fragment.FragmentScope
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.help.faq.FAQActivity
+import org.oppia.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
-/** [ViewModel] for [HelpFragment]. */
-@FragmentScope
-class HelpViewModel @Inject constructor() : ViewModel()
+/** [ObservableViewModel] for the recycler view of HelpActivity. */
+class HelpViewModel @Inject constructor() : ObservableViewModel(), HelpNavigator {
+  private var title = ""
+  private lateinit var activity: AppCompatActivity
```

Comment:
Do not you this instead make change in constructor, something like this:
`https://github.com/oppia/oppia-android/blob/3f652f5a303a506b8d87be8e4bd160828fdc7352/app/src/main/java/org/oppia/app/help/faq/FAQActivityPresenter.kt#L10`

Reply:
@rt4914
I did the changes as follows:

 ```
class HelpViewModel @Inject constructor(
  private val title: String,
  private val position: Int,
  private val activity: AppCompatActivity
) 
```

But for this, there is an error coming as follows:

`D:\IIITM STUDIES\ANDROID DEV\opensource\oppia-android\app\build\tmp\kapt3\stubs\debug\org\oppia\app\application\ApplicationComponent.java:11: error: [Dagger/MissingBinding] java.lang.String cannot be provided without an @Inject constructor or an @Provides-annotated method.
public abstract interface ApplicationComponent {
                ^
      java.lang.String is injected at
          org.oppia.app.help.HelpViewModel(title, …)
      javax.inject.Provider<org.oppia.app.help.HelpViewModel> is injected at
          org.oppia.app.viewmodel.ViewModelBridgeFactory(viewModelProvider)
      org.oppia.app.viewmodel.ViewModelBridgeFactory<org.oppia.app.help.HelpViewModel> is injected at
          org.oppia.app.viewmodel.ViewModelProvider(bridgeFactory)
      org.oppia.app.viewmodel.ViewModelProvider<org.oppia.app.help.HelpViewModel> is injected at
          org.oppia.app.help.HelpFragmentPresenter(…, viewModelProvider)
      org.oppia.app.help.HelpFragmentPresenter is injected at
          org.oppia.app.help.HelpFragment.helpFragmentPresenter
      org.oppia.app.help.HelpFragment is injected at
          org.oppia.app.fragment.FragmentComponent.inject(org.oppia.app.help.HelpFragment) [org.oppia.app.application.ApplicationComponent ? org.oppia.app.activity.ActivityComponent ? org.oppia.app.fragment.FragmentComponent]`

So what should I do here?